### PR TITLE
ful_alerts-guide-update

### PR DESCRIPTION
**Pull request by fulyauluturk**
Updated alerts guide section and the Alerts Overview page. Also some minor updates to solution specific AR pages.

### CODE REVIEW

File: docs/alerts/index.md [hunk 1]
```diff
@@ -38,3 +40,71 @@ Anomaly | yes | yes | yes
...
 
 You can manage Alert rules interactively via the UI, or you can [manage alerts via the API](../api).
+
+## Creating an Alert Rule
+
+There are multiple methods to create an alert rule. One approach is to access any App report, hover over a chart, and click on the small alert icon that appears at the top. This action will prompt a 'Create Alert Rule' dialog, automatically populating properties such as filtering and grouping based on the settings applied within that specific chart.
+
+Alternatively, you can create an alert rule by navigating to any App report and clicking on the alert icon in the actions menu at the top of the report.
+
+Additionally, you have the option to access the Alerts page through the left menu panel and select '+ New Alert Rule' from the dropdown menu located at the top right of the screen.
```
Comment (otisg) [inline docs/alerts/index.md:50]: Looking just at the diff, so not sure if the image below shows all 3 ways of creating an AR.  Aha, I see now, it's an animated GIF and it looks like it captures them all.

File: docs/alerts/index.md [hunk 2]
```diff
@@ -38,3 +40,71 @@ Anomaly | yes | yes | yes
...
+####  Group by
+
+In the Group by section you choose what data series to group on. For example, if you
+choose the `os.host` tag and for the aggregation in `Aggregation` field you choose `all separately`
+the alert will trigger for each host separately.
+
+####  Rollup by
+
+Data points sent to Sematext Cloud have a time label. If a metric is updated every 10 seconds, and you are
+looking at a 24-hour time range, you need 14,400 points to display all data points.. Rollup by is a function used to group data points in time buckets. So the metrics coming in within the specified time window of the alert (which is set in the condition tab) will be aggregated by the selected function in this section.
```
Comment (otisg) [inline docs/alerts/index.md:83]: .. ==> .

File: docs/alerts/index.md [hunk 3]
```diff
@@ -38,3 +40,71 @@ Anomaly | yes | yes | yes
...
+
+Refer to [Rollup By](../dashboards/chart-builder/#rollup-by) to learn more.
+
+### Alert Condition Tab
+
+This is where you select the [alert type](https://sematext.com/docs/alerts/#alert-types) and the priority. If you select threshold type alert you need to enter the threshold value to trigger and alert and specify a time window.
```
Comment (otisg) [inline docs/alerts/index.md:89]: If you select threshold type alert you need to enter the threshold value to trigger and alert and specify a time window. ==>
If you select threshold type alert then you need to enter the threshold value to trigger an alert and specify a time window.

File: docs/alerts/index.md [hunk 4]
```diff
@@ -38,3 +40,71 @@ Anomaly | yes | yes | yes
...
+
+#### Ignore regularly occuring spikes and dips 
+
+Where we look at historical data and try to decide if an alert fits the regular stream of spikes and dips in metrics. If it does, we won't interrupt you with notifications.
```
Comment (otisg) [inline docs/alerts/index.md:93]: Maybe this "Where" is erroneous?  "We look at..."

File: docs/alerts/index.md [hunk 5]
```diff
@@ -38,3 +40,71 @@ Anomaly | yes | yes | yes
...
+Based on the filter combination we find all previous spikes and dips from the past. Split them into group-by values (+-5% we assume as the same), find LCM (least common multiple) in the groups. This means we're trying to include the current value into our LCM groups to check if it fits any of them or not. If it fits, we assume this is a regular event and ignore it.
```
Comment (otisg) [inline docs/alerts/index.md:94]: Split them... ==> We split them?
we assume as the same ==> we treat as the same

File: docs/guide/alerts-guide.md [hunk 6]
```diff
@@ -13,26 +13,28 @@ Receiving Alerts when your software is misbehaving or underperforming is crucial
...
 
-**Threshold** and **Anomaly** Alerts can be triggered for both Monitoring and Logs Apps, while **Heartbeat** Alerts are only available for Monitoring Apps.
+![Sematext Anomaly Alerts](../images/guide/alerts-and-events/anomaly-alerts.png)
 
-Sematext lets you manage Alert Rules across your whole stack. You can use various notification channels to get alerted when critical issues occur so you can take action and resolve them. Our Alerts easily integrate with notification and chat software. 
+**Threshold** and **Anomaly** Alerts can be triggered for [Events](https://sematext.com/docs/events/) and [Monitoring](https://sematext.com/docs/alerts/creating-metrics-alerts/), [Logs](https://sematext.com/docs/alerts/creating-logs-alerts/), [Synthetics](https://sematext.com/docs/alerts/creating-synthetics-alerts/) and [Experience](https://sematext.com/docs/alerts/creating-experience-alerts/) Apps, while [**Heartbeat**](https://sematext.com/docs/alerts/creating-heartbeat-alerts/) Alerts are only available for Monitoring Apps.
```
Comment (otisg) [inline docs/guide/alerts-guide.md:26]: ... alerts can be triggered for Events -- that's not true.

File: docs/guide/alerts-guide.md [hunk 7]
```diff
@@ -13,26 +13,28 @@ Receiving Alerts when your software is misbehaving or underperforming is crucial
...
 
-- PagerDuty
-- OpsGenie
-- HipChat
-- Slack, and [many more](https://sematext.com/docs/integration/). 
+Sematext lets you manage Alert Rules across your whole stack. You can use various [notification channels](https://sematext.com/docs/alerts/alert-notifications/) to get alerted when critical issues occur so you can take action and resolve them. Our Alerts easily integrate with notification and chat software. 
+
+- [PagerDuty](https://sematext.com/docs/integration/alerts-pagerduty-integration/)
+- [OpsGenie](https://sematext.com/docs/integration/alerts-opsgenie-integration/)
+- [HipChat](https://sematext.com/docs/integration/alerts-hipchat-integration/)
```
Comment (otisg) [inline docs/guide/alerts-guide.md:32]: We can replace this with something.  HipChat is history. https://community.atlassian.com/t5/Hipchat/ct-p/hipchat
Maybe we should remove it all together.

File: docs/images/guide/alerts-and-events/anomaly-alerts.png
Comment (otisg) [inline docs/images/guide/alerts-and-events/anomaly-alerts.png:1]: Could we also capture and include a screenshot of the AD alert?  You know, the one that includes the confidence band?  So one can get a better feeling for what is an anomaly.

**Review by otisg — CHANGES_REQUESTED** [review]

File: docs/guide/alerts-guide.md (hunk 6 above)
Comment (fulyauluturk) [reply docs/guide/alerts-guide.md:26]: Aha yes. Don't know where this came from.

File: docs/images/guide/alerts-and-events/anomaly-alerts.png
Comment (fulyauluturk) [reply docs/images/guide/alerts-and-events/anomaly-alerts.png:1]: Removed, too many links there anyways.

File: docs/guide/alerts-guide.md (hunk 7 above)
Comment (fulyauluturk) [reply docs/guide/alerts-guide.md:32]: Removed, too many links there anyways.

File: docs/images/guide/alerts-and-events/anomaly-alerts.png
Comment (fulyauluturk) [reply docs/images/guide/alerts-and-events/anomaly-alerts.png:1]: Added.

**Comment by fulyauluturk** [comment]
@otisg I've also changed the placement of pipelines sensitive data, save from costs and discovery pages.

**Review by otisg — APPROVED** [review]
LGTM

**Pull request merged**